README.md

```diff
@@ -120,131 +163,166 @@ Pre-requisites
 If you are building your own version of the app, you must have your own logo to
 avoid app store conficts. Updating the logo is very simple using the [`ionic
 cordova resources`](https://ionicframework.com/docs/v3/cli/cordova/resources/)
 command.
 
 **Note**: You may have to install the [`cordova-res` package](https://github.com/ionic-team/cordova-res) for the command to work
 
-
-Troubleshooting
----
-- Make sure to use `npx ionic` and `npx cordova`. This is
-  because the setup script installs all the modules locally in a self-contained
-  environment using `npm install` and not `npm install -g`
-- Check the CI to see whether there is a known issue
-- Run the commands from the script one by one and see which fails
-    - compare the failed command with the CI logs
-- Another workaround is to delete the local environment and recreate it
-    - javascript errors: `rm -rf node_modules && npm install`
-    - native code compile errors: `rm -rf plugins && rm -rf platforms && npx cordova prepare`
-
-Beta-testing debugging
----
+## Beta-testing debugging
 If users run into problems, they have the ability to email logs to the
 maintainer. These logs are in the form of an sqlite3 database, so they have to
 be opened using `sqlite3`. Alternatively, you can export it to a csv with
 dates using the `bin/csv_export_add_date.py` script.
 
 ```
 <download the log file>
-$ mv ~/Downloads/loggerDB /tmp/logger.<issue>
-$ pwd
+mv ~/Downloads/loggerDB /tmp/logger.<issue>
+pwd
 .../e-mission-phone
-$ python bin/csv_export_add_date.py /tmp/loggerDB.<issue>
-$ less /tmp/loggerDB.<issue>.withdate.log
+python bin/csv_export_add_date.py /tmp/loggerDB.<issue>
+less /tmp/loggerDB.<issue>.withdate.log
 ```
 
-Contributing
----
-
-Add the main repo as upstream
-
-    $ git remote add upstream https://github.com/covid19database/phone-app.git
-
-Create a new branch (IMPORTANT). Please do not submit pull requests from master
+## Contributing
 
-    $ git checkout -b mybranch
 
-Make changes to the branch and commit them
-
-    $ git commit
-
-Push the changes to your local fork
+1. Add the main repo as upstream
+```
+git remote add upstream https://github.com/e-mission/e-mission-phone
+```
+2. Create a new branch (IMPORTANT). Please do not submit pull requests from master
+```
+git checkout -b <branch>
+```
+3. Make changes to the branch and commit them
+```
+git commit
+```
+4. Push the changes to your local fork
+```
+git push origin <branch>
+```
+5. Generate a pull request from the UI
 
-    $ git push origin mybranch
+<br>
 
-Generate a pull request from the UI
+__\*__Address my review comments__\*__
 
-Address my review comments
+Once I merge the pull request :smiley: :tada:, pull the changes to your fork and delete the branch
+```
+git checkout master
+```
+```
+git pull upstream master
+```
+```
+git push origin master
+```
+```
+git branch -d <branch>
+```
 
-Once I merge the pull request, pull the changes to your fork and delete the branch
+---
+### Troubleshooting
```

Comment:
This is troubleshooting for the native build - why has it been moved to the bottom?

Reply:
This has not been addressed